sale_cancel_reason/__manifest__.py

```diff
@@ -1,20 +1,24 @@
 # -*- coding: utf-8 -*-
 # Copyright 2013 Guewen Baconnier, Camptocamp SA
+# Copyright 2019 Victor M.M. Torres, Tecnativa SL
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-{'name': 'Sale Cancel Reason',
- 'version': '10.0.1.0.0',
- 'author': "Camptocamp,Odoo Community Association (OCA)",
- 'category': 'Sale',
- 'license': 'AGPL-3',
- 'complexity': 'normal',
- 'website': "http://www.camptocamp.com",
- 'depends': ['sale',
-             ],
- 'data': ['wizard/cancel_reason_view.xml',
-          'view/sale_view.xml',
-          'security/ir.model.access.csv',
-          'data/sale_order_cancel_reason.xml',
-          ],
- 'auto_install': False,
- 'installable': True,
- }
+{
+    'name': 'Sale Cancel Reason',
+    'version': '12.0.1.0.0',
+    'author': 'Camptocamp,'
+              'Tecnativa,'
```

Comment:
@victormmtorres Maybe 'just' a migration does not make you as 'author'

Reply:
I agree in this case. Sorry for not seeing it. Please only add you in contributors.

Reply:
Changes done